server/routes/aboutPop/locale.json

```diff
@@ -1,6 +1,11 @@
 {
   "en": {
-    "title": "About {POP_NAME}",
+    "page": {
+      "title": "About {{ subject.givenName }} - {{ applicationName }}"
```

Comment:
This should just say `About - {{ applicationName }}`

Reply:
Resolved https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/113/commits/b4b6e2ceaf7c4fe8234609879588600eb174c1b4